app/javascript/packs/application.js

```diff
@@ -15,3 +15,25 @@ require("channels")
 //
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
+
+const app = require('express')();
```

Comment:
Why are we using express here? Express is relatively the same thing as Ruby on Rails. We don't need it to make the app work.